README.md

```diff
@@ -79,12 +79,19 @@ Now you can render React components from your views or your controllers.
 
 ### Rendering from a view
 
-Use the `react_component` helper:
+Use the `react_component` helper. The first argument is your component's name, the second one is the `props`:
 
 ```erb
 <%= react_component('Hello', name: 'React') %>
 ```
 
+You can pass a `tag` argument to change render the React component in another tag than the default `div`. All other arguments will be passed to `content_tag`:
```

Comment:
You can pass a `tag` argument to render the React component in a tag other than the default `div`